big_tests/tests/mam_SUITE.erl

```diff
@@ -240,9 +240,12 @@ basic_groups() ->
     [
      {mam_all, [parallel],
            [{mam04, [parallel], mam_cases() ++ [retrieve_form_fields] ++ text_search_cases()},
```

Comment:
I think we are duplicating the group here, instead of adding tests to mam04.

Reply:
Oh, indeed, thanks for the catch! I will amend the latest commit.